inference-engine/tests/ngraph_helpers/ngraph_functions/src/squeeze_unsqueeze.cpp

```diff
@@ -23,5 +23,10 @@ std::shared_ptr<ngraph::Node> makeSqueezeUnsqueeze(const ngraph::Output<Node> &i
             throw std::logic_error("Unsupported operation type");
     }
 }
+
+std::shared_ptr<ngraph::Node> makeSqueezeNoAxes(const ngraph::Output<Node> &in,
```

Comment:
Since this function is just one line, what about remove it and create the op inside Setup method?

Otherwise, shouldn't you use type param to create the op?

Reply:
Done